.github/workflows/connect-dev-release-test.yml

```diff
@@ -108,3 +107,62 @@ jobs:
             packages/connect-examples/webextension-mv3/build-legacy
             packages/connect-examples/webextension-mv3-sw/build
             packages/connect-explorer/build-webextension
+
+  methods:
+    needs: [build-deploy]
```

Comment:
needs means that it waits for this job and possibly has access to its artifacts, right?

Reply:
A job with a needs reference will only start after all the jobs it depends on have completed successfully. But it does not mean it will have access to it's artifacts. It is possible to upload artifacts and then download it in other job.

In this case `build-deploy` is deploying connect-explorer and it's dependencies (popup...) to https://dev.suite.sldev.cz/connect/<branch-name> and that URL is used by the tests.